roles/openshift_logging/templates/kibana.j2

```diff
@@ -67,10 +71,8 @@ spec:
 {% if kibana_proxy_cpu_limit is not none %}
               cpu: "{{kibana_proxy_cpu_limit}}"
 {% endif %}
-{% if kibana_proxy_memory_limit is not none %}
               memory: "{{kibana_proxy_memory_limit}}"
```

Comment:
Should we also have a default value specified here since we don't have one when this template is used (e.g. what if a user specifies the value as null)